### DISCUSSION TITLE
Remove Special Repository Status

### DISCUSSION THREAD

**Discussion by toddcf** [community/community#30987] [Other Feature Feedback, Questions, & Ideas]
Hello,

Once a repository has been made a "special repository," does anyone know how to undo this?  (I want my repository to go back to just being a regular repository, without its README linking to my public profile.)

Thanks!

**Comment by radar-blip** [comment]
You can delete it in settings I believe.
Just open the repo, click on settings (at the top of repo) and scroll down to danger zone. Delete.
Or delete the README.md which it seems you have already done. 👍🏻

**Reply by toddcf** [reply]
Thank you.  Yes, I deleted the README, but I don't want to delete the repo.  I want to keep the repo, remove "special repository" status, and then be able to add a README back onto it (without it appearing on my public profile).

**Reply by radar-blip** [reply]
Yes, I understand now. But I dont have a solution for you. 
Maybe someone else might have an idea. 
Good Luck!

**Comment by hariprasd** [comment]
> Hello,
> 
> Once a repository has been made a "special repository," does anyone know how to undo this? (I want my repository to go back to just being a regular repository, without its README linking to my public profile.)
> 
> Thanks!

Github uses the same name for the repo as your username, So if you rename the repo, It wont be special anymore

**Comment by SaptarshiSarkar12** [comment]
**Change** the name of the repository to any other name of your choice.

**Reply by toddcf** [reply]
Thanks for your ideas, everyone.  I've changed the repo name as a temporary workaround, and submitted a ticket to GitHub to see if there's a way to go back to using my original repo name without it reverting to a special repository.  If there's a way, I'll post it here.

**Reply by SaptarshiSarkar12** [reply]
@toddcf Please mark the correct answers to prevent misguides.
